yt_dlp/extractor/magellantv.py

```diff
@@ -46,7 +47,13 @@ def _real_extract(self, url):
         data = traverse_obj(self._search_nextjs_data(webpage, video_id), (
             'props', 'pageProps', 'reactContext',
             (('video', 'detail'), ('series', 'currentEpisode')), {dict}), get_all=False)
-        formats, subtitles = self._extract_m3u8_formats_and_subtitles(data['jwpVideoUrl'], video_id)
+        formats = []
+        subtitles = {}
+        for url in traverse_obj(data, ((('manifests', ..., 'hls'), 'jwp_video_url'), {url_or_none})):
```

Comment:
Is this getting duplicate formats? 

I think `subs` only come from `v2` in the Ellipsis: hence it should be enough to use that if present and any other (probably, `v1` which I found to give the same as the `jwp_video_url`) if not.

Reply:
v1 sometimes contains hevc variants.

Reply:
Does something eliminate the formats extracted from the `.jwp_video_url` value if `.v1.hls` has the same value?